addons/account/models/account_reconcile_model.py

```diff
@@ -231,134 +230,121 @@ def _get_taxes_move_lines_dict(self, tax, base_line_dict):
                 'account_id': tax_res['account_id'] or base_line_dict['account_id'],
                 'name': tax_res['name'],
                 'partner_id': base_line_dict.get('partner_id'),
-                'debit': tax_res['amount'] > 0 and tax_res['amount'] or 0,
-                'credit': tax_res['amount'] < 0 and -tax_res['amount'] or 0,
+                'balance': tax_res['amount'],
                 'analytic_account_id': tax.analytic and base_line_dict['analytic_account_id'],
                 'analytic_tag_ids': tax.analytic and base_line_dict['analytic_tag_ids'],
                 'tax_exigible': tax_res['tax_exigibility'],
                 'tax_repartition_line_id': tax_res['tax_repartition_line_id'],
                 'tax_ids': [(6, 0, tax_res['tax_ids'])],
                 'tax_tag_ids': [(6, 0, tax_res['tag_ids'])],
+                'tax_tag_ids': [(6, 0, tax_res['tag_ids'])],
+                'currency_id': False,
```

Comment:
??

Reply:
Because this dictionary is passed to the reconcile method in the bank statement line. Then, `currency_id` must be specified to tell on which currency `balance` is expressed.